### PR TITLE
Book

### PR DESCRIPTION
**Pull request by joredudiaz**
# Summary
- book model created
# Trello card
https://trello.com/c/Hv3saHbW

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -68,4 +68,7 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 
-gem 'devise'
\ No newline at end of file
+gem 'devise'
```
Comment (holywyvern) [inline Gemfile]: This is from the last PR.

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
-  devise_for :users
+  mount_devise_token_auth_for 'User', at: 'auth'
```
Comment (holywyvern) [inline config/routes.rb]: This is also from last PR

Comment (holywyvern) [reply config/routes.rb]: Are you forgetting a rebase or a pull/push ?

Comment (joredudiaz) [reply config/routes.rb]: I do branches from the last branch that it depends. I don't do pull/push and I do rebase just when you accept my PR.

File: db/migrate/20181025205222_create_books.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :genre, :null => false, :default => ""
```
Comment (holywyvern) [inline db/migrate/20181025205222_create_books.rb]: remember the preffered style for Hash of symbol keys is `t.string :genre, null: false, default: ""`

**Review by holywyvern — CHANGES_REQUESTED** [review]

**Review by leoncalermo — COMMENTED** [review]
Please, make more specific commits messages, the idea it's a brief explanation of the changes in the commit, with `UPDATE`  and `CREATE`, it's hard to track your work in each commit

**Review by leoncalermo — COMMENTED** [review]
I think it's Ok but you have Merge Conflicts fix them and we are ready to merge 🎉

**Pull request merged**